cluster-monitoring/templates/opsgenie-heartbeat-proxy-deployment.yaml

```diff
@@ -18,34 +15,34 @@ spec:
         release: {{ .Release.Name }}
     spec:
       containers:
-        - name: {{ template "opsgenieHeartbeatProxy.fullname" . }}
-          image: "{{ .Values.opsgenieHeartbeatProxy.image }}:{{ .Values.opsgenieHeartbeatProxy.imageTag }}"
-          imagePullPolicy: {{ default "" .Values.opsgenieHeartbeatProxy.imagePullPolicy | quote }}
-          env:
-            - name: OPSGENIE_API_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: {{ template "opsgenieHeartbeatProxy.fullname" . }}
-                  key: opsgenie_api_key
-            - name: HEARTBEAT_NAME
-              valueFrom:
-                configMapKeyRef:
-                  name: {{ template "opsgenieHeartbeatProxy.fullname" . }}
-                  key: heartbeat_name
-          ports:
-            - name: opsgenie-proxy
-              containerPort: {{ .Values.opsgenieHeartbeatProxy.port }}
-          livenessProbe:
-            tcpSocket:
-              port: opsgenie-proxy
-            initialDelaySeconds: 10
-            timeoutSeconds: 5
-          readinessProbe:
-            tcpSocket:
-              port: opsgenie-proxy
-            initialDelaySeconds: 10
-            timeoutSeconds: 5
-          resources:
+      - name: {{ template "opsgenieHeartbeatProxy.fullname" . }}
```

Comment:
![](https://media.giphy.com/media/cPKWZB2aaB3rO/giphy.gif)

Reply:
It looks better IMHO 😅 .

Also k8s docs follow the same indentation, eg https://kubernetes.io/docs/concepts/workloads/controllers/deployment/ ;)